zhaquirks/tuya/ts0601_valve.py

```diff
@@ -568,3 +569,74 @@ class GiexIrrigationStatus(t.enum8):
     .skip_configuration()
     .add_to_registry()
 )
+
+
+class RoyalGardineerWeatherDelay(t.enum8):
+    """Royal Gardineer Irrigation Valve weather delay enum."""
+
+    Disabled = 0x00
+    Delayed_24h = 0x01
+    Delayed_48h = 0x02
+    Delayed_72h = 0x03
+
+
+class RoyalGardineerTimerState(t.enum8):
+    """Royal Gardineer Irrigation Valve timer state enum."""
+
+    Disabled = 0x00
+    Active = 0x01
+    Enabled = 0x02
+
+
+(
+    TuyaQuirkBuilder("_TZE200_2wg5qrjy", "TS0601")
+    .tuya_onoff(dp_id=1)
+    # Should be TuyaPowerConfigurationCluster2AA, but it is broken at this time.
+    .tuya_battery(dp_id=7, power_cfg=TuyaPowerConfigurationCluster2AA)
+    # Might need a converter: x // 10
+    .tuya_metering(dp_id=5)
+    # Timer time left/remaining (raw value in seconds).
+    .tuya_number(
+        dp_id=11,
+        attribute_name="timer_time_left",
+        type=t.uint32_t,
+        min_value=1,
+        max_value=600,
+        step=1,
+        multiplier=1 / 60,
+        unit=UnitOfTime.MINUTES,
+        translation_key="timer_time_left",
+        fallback_name="Timer time left",
+    )
```

Comment:
@mrrstux Just to confirm, this is adjustable by the user? Or should it be a sensor?

Reply:
Ok, looking at Z2M, that this value is adjustable. It just has two "exposes": one only readable, one only writeable.